ruby/lib/minitest/queue/runner.rb

```diff
@@ -19,15 +18,16 @@ def self.invoke(argv)
       end
 
       def initialize(argv)
-        @queue_config = CI::Queue::Configuration.from_env(ENV)
-        @command, @argv = parse(argv)
+        @queue_config = CI::Queue::Configuration.new(ENV, argv).config
+        @command = argv[0]
+        @files = argv[1..-1]
       end
```

Comment:
The old code looks more readable. Don't you think so?

Reply:
I can put it behind a new method. However, the old method is mutating the argv object, which is dangerous.